src/click/core.py

```diff
@@ -2420,6 +2420,8 @@ class Option(Parameter):
                          help page. Normally, defaults are not shown. If this
                          value is a string, it shows the string instead of the
                          value. This is particularly useful for dynamic options.
+                         For single option boolean flags, the default remains hidden
```

Comment:
Looks like this line sticks out more. Docstrings and docs pages are wrapped at 72 characters (code is 88).